package.json

```diff
@@ -0,0 +1,92 @@
+{
+    "name": "ilib-localedata",
+    "version": "1.0.0",
+    "main": "./lib/index.js",
+    "description": "Load and cache iLib locale data",
+    "keywords": [
+        "internationalization",
+        "i18n",
+        "localization",
+        "l10n",
+        "globalization",
+        "g11n",
+        "date",
+        "time",
+        "format",
+        "locale",
+        "translation",
+        "localedata"
+    ],
+    "homepage": "https://github.com/iLib-js/ilib-localedata",
+    "bugs": "https://github.com/iLib-js/ilib-localedata/issues",
+    "email": "marketing@translationcircle.com",
+    "license": "Apache-2.0",
+    "author": {
+        "name": "Edwin Hoogerbeets",
+        "web": "http://www.translationcircle.com/",
+        "email": "edwin@translationcircle.com"
+    },
+    "contributors": [
+        {
+            "name": "Edwin Hoogerbeets",
+            "email": "ehoogerbeets@gmail.com"
+        }
+    ],
+    "files": [
+        "src",
+        "lib",
+        "docs",
+        "README.md",
+        "LICENSE"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git@github.com:iLib-js/ilib-localedata.git"
+    },
+    "scripts": {
+        "build": "npm run build:prod",
+        "build:prod": "grunt babel --mode=prod",
+        "build:dev": "grunt babel --mode=dev",
+        "build:web": "webpack-cli",
+        "build:test": "webpack-cli --config webpack-test.config.js",
+        "dist": "npm run doc ; npm run build:prod ; npm run build:web; npm pack",
+        "test": "LANG=en_US.UTF8 npm run build:dev ; node test/testSuite.js",
+        "testall": "LANG=en_US.UTF8 npm run test; npm run build:test ; open-cli ./test/testSuite.html ; open-cli ./test/testSuite.html -- firefox",
```

Comment:
Why don't we keep only the valid command at this moment? It will help to check the implementation status. when it's really working, we'd better add it later.